### PR TITLE
Made ajax dependencies send url as name

### PR DESCRIPTION
**Pull request by madeleinefletcher**
Link to bug 484887 in VSO

### CODE REVIEW

**Comment by AlexBulankou** [comment]
Looks :shipit: to me, but let's have @MaxShehovtsov double check

File: JavaScript/JavaScriptSDK/Util.ts [hunk 1]
```diff
@@ -397,5 +397,15 @@ module Microsoft.ApplicationInsights {
 
             return result;
         }
+
```
Comment (MaxShehovtsov) [inline JavaScript/JavaScriptSDK/Util.ts]: Should getHostName be removed (or is it still used)?

Comment (madeleinefletcher) [inline JavaScript/JavaScriptSDK/Util.ts]: I think it can be removed. do you want me to submit a pull request for that?

Comment (MaxShehovtsov) [inline JavaScript/JavaScriptSDK/Util.ts]: that would be great, let's try to avoid the dead code :)

**Pull request merged**